tests/RandomTest.php

```diff
@@ -4,7 +4,7 @@
 
 use atk4\data\Model;
 use atk4\data\Persistence;
-use atk4\data\Persistence_SQL;
+use atk4\data\Persistence\SQL as Persistence_SQL;
```

Comment:
we should leave class Persistence_SQL extends Persistence\SQL for compatibility, so this shouldn't be necessary.

Reply:
We leave these old classes for compatibility for now, but should remove them at some point (in next major release anyway), so it's better to prepare our tests already for that.